packaging/hudi-spark-bundle/pom.xml

```diff
@@ -441,5 +441,35 @@
         <spark.bundle.hive.shade.prefix>org.apache.hudi.</spark.bundle.hive.shade.prefix>
       </properties>
     </profile>
+    <profile>
+      <id>hudi-platform-service</id>
```

Comment:
should this also be added to flink bundle?

Reply:
fix